node/cn/gasprice/gasprice.go

```diff
@@ -339,6 +331,33 @@ func (oracle *Oracle) getBlockValues(ctx context.Context, blockNum uint64, limit
 	}
 }
 
+func (oracle *Oracle) isRelaxedNetwork(header *types.Header) bool {
```

Comment:
```suggestion
// isRelaxedNetwork returns true if the current network congestion is low to the point paying any tip is unnecessary. It returns true when the head block is after Magma fork and the next base fee is at the lower bound. 
func (oracle *Oracle) isRelaxedNetwork(header *types.Header) bool {
```

Reply:
Fixed in [407d41e](https://github.com/kaiachain/kaia/pull/44/commits/407d41e1329b657a20ac7a0383952dd51d89d42a)